tests/clone.rs

```diff
@@ -1,12 +1,14 @@
 #![allow(dead_code)]
 
+use std::marker::PhantomData;
```

Comment:
```suggestion
use core::marker::PhantomData;
```

We should test for `#[no_std]` in the future too.